docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+
+### Commitment mechanism
+
+**CIC Always Committed to by OP's Signature:**
+The CIC protected header contains the identity's public key and associated metadata. To bind the identity's public key to the ID Token, the CIC is always committed to in the payload or the OP's protected header. Why payload or OP's protected header? Because the OP's signature signs both the payload and the protected header. This means the binding between identity and public key is made by the party that authenticated the identity and determined the identity is are who they say they are.
+
+In our PK Token zoo below we describe four different mechanisms that the CIC can be committed to under the OP's signature:
+
+| Commitment Type               | Commitment Location  | Claim name |
+| -------------                 |--------------        |--------------|
+| Nonce-Commitment              | `payload`            | `nonce`      |
+| Audience-Commitment           | `payload`            | `aud`        |
+| GQ-Commitment                 | `OP-protected header`| `CIC`        |
+| ZK-Commitment                 | `OP-protected header`| `CIC`        |
+
+### Types of Signatures in a PK Token
+
+#### OP (OpenID Provider) Signature
+
+This is signature of the OpenID Provider that created the payload and signed it to create the ID Token. This signature is responsible for binding the identity claims in the payload to that identity's public key.
+
+Typically OP signature/protected header along with the payload is the ID Token. For some more advanced forms of PK Tokens such as GQ or ZK PK Tokens, we transform OP signature and protected header. In these cases, you can no longer recover the ID Token from the PK Token, but even when we alter the OP Signature and protected header in this way, we don't break the cryptographic binding between payload and OpenID Provider that the signature provides.
+
+Some OpenID Providers do not specify `typ`. To get around this we classify a signature as being from the OpenID Provider if either `typ=JWT` or `typ` is not defined.
+
+#### CIC (Client-Instance Claims) Signature
+
+The Client-Instance is the identity's OpenPubkey client. The CIC are the claims made about the identity by this client. The CIC signature performs two functions. First, it provides the needed data to allow verifiers to check that an ID Token has a binding to the user's public key by opening the commitment. Second, it functions as a [Proof-of-Possession](https://csrc.nist.gov/glossary/term/proof_of_possession) showing that the identity's knows the signing key associated with the public key `upk`. This Proof-of-Possession prevents rogue key attacks in which a attackers associates their identity with another identity's public key and the attempts to claim that they produced signatures produced by the other identity.
```

Comment:
Spell check: "in which **an attacker** associates their identity with another identity's public key and the**n** attempts to claim that they produced signatures **from** the other identity." (modifications in bold)